iroha/src/torii/utils.rs

```diff
@@ -0,0 +1,96 @@
+use std::convert::TryInto;
```

Comment:
Generally it is not a good practice to group code into `utils` module. There is not much code here so it can be directly in torii. If there were more code than they could be subdivided into other modules with corresponding names by their function/domain. 

Creating a utils module violates `High Cohesion and Low Coupling` principle.

Reply:
It is not tied up to torii, it can be generally used with warp, so thats why it is in separate module.

Reply:
But the only place where we will need it is torii... At least in this form. For example if later we will use scale with warp in some other crate we can make a crate warp_scale for this.

Reply:
Looked up for `high cohesion and low coupling`:
- Seems like this module is exactly low coupled as it doesnt use anything from torii itself.
- As for high cohesion everything in the module tied up by context

Reply:
Then it should be called differently :) I mean `utils` naming generally leads to low cohesion - as many different unrelated utils appear there.

Reply:
Name doesnt lead to anything. I just didnt found a better name here. May be you can suggest?

Reply:
Ok, let's keep utils